src/universal/components/AnonymousReflectionCard/AnonymousReflectionCard.js

```diff
@@ -5,87 +5,119 @@
  * @flow
  */
 // $FlowFixMe
-import {ContentState, EditorState} from 'draft-js';
+import {ContentState, EditorState, SelectionState} from 'draft-js';
 import React, {Component} from 'react';
-import styled from 'react-emotion';
+import {createFragmentContainer} from 'react-relay';
 
-import EditorInputWrapper from 'universal/components/EditorInputWrapper';
-import ReflectionCardWrapper from 'universal/components/ReflectionCardWrapper/ReflectionCardWrapper';
+import type {AnonymousReflectionCard_reflection as Reflection} from './__generated__/AnonymousReflectionCard_reflection.graphql';
+import type {AnonymousReflectionCard_meeting as Meeting} from './__generated__/AnonymousReflectionCard_meeting.graphql';
+import reactLifecyclesCompat from 'react-lifecycles-compat';
+import ReflectionEditorWrapper from 'universal/components/ReflectionEditorWrapper';
+import {makeContentWithEntity} from 'universal/utils/draftjs/completeEnitity';
+import anonymousReflectionDecorators from 'universal/components/TaskEditor/anonymousReflectionDecorators';
+import styled from 'react-emotion';
+import ui from 'universal/styles/ui';
 
 type Props = {
-  contentState: ContentState,
-  isEditing?: boolean
+  meeting: Meeting,
+  reflection: Reflection,
 };
 
 type State = {
-  obfuscatedContentState: ContentState
+  editorState: EditorState,
+  content: string,
+  isBlurred: boolean,
+  isEditing: boolean
 };
 
-const randomChar = () => String.fromCharCode(97 + Math.floor(Math.random() * 26));
-
-// Given a string, returns another string which has the same "shape", e.g.
-// same punctuation and word length, but with completely random characters.
-const obfuscate = (content: string): string => (
```

Comment:
using a random approach means the string changes every time. it's distracting to see something blurred changing a bunch, takes away from the meeting.

Reply:
Good call. Amazing how much thought you put into this.